views/Users/List/List.module.scss

```diff
@@ -0,0 +1,13 @@
+@use 'colors';
```

Comment:
why not just `import`?

Reply:
I saw in sass documentation that @import is going to be deprecated and replaced for @use.

Reply:
https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline

Here it mentions that it won't be changed soon

> July 2022: In light of the fact that LibSass was deprecated before ever adding support for the new module system, the timeline for deprecating and removing @import has been pushed back. We now intend to wait until 80% of users are using Dart Sass (measured by npm downloads) before deprecating @import, and wait at least a year after that and likely more before removing it entirely.

Reply:
Oh ok, I didn't know about that. Fixed.